routes/collection/index.tsx

```diff
@@ -54,25 +65,165 @@ export default function Collection(props: CollectionPageProps) {
     pages,
     page_size,
     filterBy,
-    sortBy,
-    selectedTab,
+    stamps_src721 = [],
+    stamps_posh = [],
   } = props.data;
 
+  const SectionsCollections: StampSectionProps[] = [
+    {
+      title: "FRESH POSH STAMPS",
+      type: "posh",
+      stamps: stamps_posh,
+      layout: "grid",
+      showDetails: false,
+    },
+    {
+      title: "RECENT RECURSIVE",
+      type: "recursive",
+      stamps: stamps_src721,
+      layout: "row",
+      showDetails: false,
+    },
+  ];
+
   return (
-    <div class="flex flex-col gap-8">
-      <CollectionHeader
-        filterBy={filterBy}
-        sortBy={sortBy}
-        selectedTab={selectedTab}
-      />
-      <CollectionList collections={collections} />
-      <Pagination
-        page={page}
-        pages={pages}
-        page_size={page_size}
-        type={"collection"}
-        data_length={collections.length}
-      />
+    <div className="text-[#CCCCCC] flex flex-col gap-36">
+      <section className="flex flex-col gap-36">
+        <div className="flex flex-col gap-6">
+          <h1 className="text-6xl purple-gradient1 font-black">
+            COLLECTIONS
+          </h1>
+          <div class="flex flex-col gap-12">
+            <CollectionSection
+              key={SectionsCollections[0].type}
+              {...SectionsCollections[0]}
+            />
+          </div>
+        </div>
+        <div className="flex flex-col gap-6">
+          <h2 className="gray-gradient3 text-6xl font-black">
+            ESPECIALLY POSH
+            <h3 className="text-5xl font-extralight">
+              STAMP COLLECTIONS
+            </h3>
+          </h2>
+          <div className="grid grid-cols-2 tablet:grid-cols-3 gap-6">
+            {collections.slice(0, 3).map((collection) => (
+              <CollectionListCard
+                key={collection.collection_id}
+                collection={collection}
+              />
+            ))}
+          </div>
+          <div>
+            <h3 className="text-5xl font-extralight">NAMED ASSETS</h3>
```

Comment:
was this validated against the figma design? i'm surprised there aren't different font sizes at the different breakpoints